modules/oracledb/charts.go

```diff
@@ -0,0 +1,108 @@
+package oracledb
+
+import "github.com/netdata/go.d.plugin/modules"
+
+type (
+	// Charts is an alias for modules.Charts
+	Charts = modules.Charts
+	// Dims is an alias for modules.Dims
+	Dims = modules.Dims
+)
+
+var charts = Charts{
+	{
+		ID:    "processes",
+		Title: "Processes",
+		Ctx:   "oracledb.processes",
+		Dims: Dims{
+			{ID: "processes", Name: "processes"},
+		},
+	},
+	{
+		ID:    "sessions",
+		Title: "Total Sessions",
+		Ctx:   "oracledb.sessions",
+		Dims: Dims{
+			{ID: "sessions_total", Name: "sessions_total"},
+			{ID: "sessions_active", Name: "sessions_active"},
+			{ID: "sessions_inactive", Name: "sessions_inactive"},
```

Comment:
i don't like that one chart contains total and specific types of connections.

My suggestion (or):
 * move `total` to a separate chart
 * remove total, change chart algorithm to `stacked` (total will be `active` + `inactive`)

And, this is true for **all charts**:
 1. don't copy dimension id to `name`

name
> is the name that will be presented to the user instead of id in type.id. This means that only the id part of type.id is changed. When a name has been given, the chart is index (and can be referred) as both type.id and type.name. You can set name to '', or null, or (null) to disable it.

Should be:

```go
			{ID: "sessions_active", Name: "active"},
			{ID: "sessions_inactive", Name: "inactive"}
```

 2. please add `Units` to all charts.

```go
		ID:    "sessions",
		Title: "Sessions",
		Ctx:   "oracledb.sessions",
         	Units:   "sessions",
		Dims: Dims{
			{ID: "sessions_total", Name: "total"},
			{ID: "sessions_active", Name: "active"},
			{ID: "sessions_inactive", Name: "sessions_inactive"},
```

[Dimension algorithm](https://docs.netdata.cloud/collectors/plugins.d/#dimension) by default is `absolute`. If you need `incremental` you have to set it explicitly

Reply:
Same for `processes`? the dim should be: `Dim {  ID: "processes", Name: "processes", Algo: modules.Incremental }` for example?

Reply:
The dim should be: `{  ID: "sessions_active", Name: "active", Algo: modules.Incremental }` for example? We should start writing some documentation for these things for the go.d.plugin...

Reply:
it's netdata api, it's[ documented](https://docs.netdata.cloud/collectors/plugins.d/#external-plugins-api)